demos/node-classification/graphsage/graphsage-pubmed-inductive-node-classification-example.ipynb

```diff
@@ -191,31 +140,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "(19717, 501)"
+       "19717"
       ]
      },
-     "execution_count": 6,
+     "execution_count": 5,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "node_data.shape"
+    "len(labels)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [],
    "source": [
-    "node_data_sampled = node_data.sample(frac=0.8, replace=False, random_state=101)"
+    "labels_sampled = labels.sample(frac=0.8, replace=False, random_state=101)"
```

Comment:
You have renamed node_data_sampled to labels_sampled but the comment following this code still references node_data_sampled. Honestly, I thought the old name was more accurate since what is sampled here are nodes not labels.

Reply:
I think this new name is accurate: the return value is a series with contents like:

```
pid
16221917    3
18591404    2
16424231    2
9925350     3
17448371    3
           ..
18579418    1
12819907    3
15142919    3
18728230    1
8786022     1
Name: label, Length: 15774, dtype: int64
```

That is, an index called `pid`, with values 1, 2, 3.

This is matching the old form, where the "`node_data`" in the name was referring to the actual node features since it was sampling whole rows of the DataFrame, not just the node IDs in the index. The sampling of node IDs is implicit in both old and new forms, as the index of the resulting structure.

You're 100% right about the following comment, though.